test/integration/css/fixtures/with-styled-jsx/pages/index.js

```diff
@@ -0,0 +1,12 @@
+export default function Home () {
+  return (
+    <>
+      <div className='my-text'>This text should be red.</div>
```

Comment:
```suggestion
      <div className='my-text'>This text should be green.</div>
```
doesn't this check to make sure it's green?